python_modules/dagster/dagster/_core/definitions/freshness_policy.py

```diff
@@ -34,12 +34,29 @@ class FreshnessPolicy(
     cron_schedule is set), the data used to produce the current version of this asset must be no
     older than `maximum_lag_minutes` before the most recent available data.
 
-    Examples:
+    The freshness status of assets with policies defined will be visible in the UI. If you are using
+    an asset reconciliation sensor, this sensor will kick off runs to help keep your assets up to
+    date with respect to their FreshnessPolicy.
+
+    Args:
+        maximum_lag_minutes (float): An upper bound for how old the data contained within this
```

Comment:
Is "relative to the current time" accurate, if you're using also using `cron_schedule`? Could maybe just leave that clause out?